src/ORM/README.md

```diff
@@ -0,0 +1,112 @@
+# CakePHP ORM
+
+The CakePHP ORM provides a powerful and flexible way to work with relational
+databases. Using a datamapper pattern the ORM allows you to manipulate data as
+entities allowing you to create expressive domain layers in your applications.
+
+## Connecting to the Database
+
+The CakePHP ORM is compatible with:
+
+* MySQL 5.1+
```

Comment:
The ORM doesn't generate and SQL that won't work on 5.1 so we can 'support' it until it is a problem.